### ISSUE TITLE
The links of the "Additional Examples" section in the docs are broken

### ISSUE THREAD

**Issue by nicolasgandrade** [expressjs/expressjs.com#1716]
### Description

Currently, all the 4 links under the "Additional Examples" section in the docs are not working. All those links redirect to legacy folders that do not exist anymore in the Prisma project.

URL: https://expressjs.com/en/starter/examples.html#additional-examples

### Expectations

The links should redirect the user to the existing folder in the Prisma project.

**Comment by nicolasgandrade** [comment]
I can replace the old links with the new ones. My only concerns are about the examples using JavaScript and GraphQL, I couldn't find them in the current version of the Prisma repository.

Should we remove that item from the docs?

**Comment by bjohansebas** [comment]
Hi @nicolasgandrade, PRs are always welcome

> Should we remove that item from the docs?

Yes, if there is no replacement, it should be removed.

**Comment by nicolasgandrade** [comment]
Right, I'll get the updated ones and open a PR fixing that. Thanks for the response

**Comment by bjohansebas** [comment]
fixed by #1717
